backend/de.metas.manufacturing/src/main/java/org/eevolution/productioncandidate/service/CreateOrderCandidateCommand.java

```diff
@@ -87,6 +88,7 @@ public I_PP_Order_Candidate execute()
 		ppOrderCandidateRecord.setC_UOM_ID(qtyRounded.getUomId().getRepoId());
 
 		ppOrderCandidateRecord.setC_OrderLine_ID(OrderLineId.toRepoId(request.getSalesOrderLineId()));
+		ppOrderCandidateRecord.setM_ShipmentSchedule_ID(NumberUtils.asInt(request.getShipmentScheduleId(), -1));
```

Comment:
same (pls use static ShipmentScheduleId.toRepoId for the sake of least surprise)